tests/_core/reindex.yaml

```diff
@@ -0,0 +1,212 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test reindex.
+epilogues:
+  - path: /movies
+    method: DELETE
+    status: [200, 404]
+  - path: /films
+    method: DELETE
+    status: [200, 404]
+  - path: /videos
+    method: DELETE
```

Comment:
we can shorten this with
```yaml
  - path: /movies,films,videos
    method: DELETE
    status: [200, 404]
```

Reply:
No, because then if 1 of these doesn't exist the entire request fails.